### PR TITLE
new tracker - EniaHD

### LINKED ISSUE #61: Add tracker EniaHD [RUS]
Просьба добавить новый трекер https://eniahd.com

### PR DESCRIPTION
**Pull request by NecroKote**
this MR adds new tracker EniaHD (eniahd.com, eniatv.com) to torrt
closes #61

### CODE REVIEW

File: tests/trackers/datafixtures/eniahd.html [hunk 1]
```diff
@@ -0,0 +1,1234 @@
+<!DOCTYPE html>
```
Comment (idlesign) [inline tests/trackers/datafixtures/eniahd.html:1]: Let's keep datafixtures minimal. 
Please cut out unimportant stuff from html (see nnm fixture for example).

**Review by idlesign — COMMENTED** [review]
Thank you. Almost ready to merge.
Just one thing, see below.

File: tests/trackers/datafixtures/eniahd.html (hunk 1 above)
Comment (NecroKote) [reply tests/trackers/datafixtures/eniahd.html:1]: I've removed all non-related stuff from the HTML.
401 insertions(+), 1224 deletions(-)

hope it helps

**Comment by idlesign** [comment]
Thank you. Merged.

**Pull request merged**